docs/sources/get-started/labels/non-indexed-labels.md

```diff
@@ -0,0 +1,50 @@
+---
+menuTitle: Non-indexed labels
+title: What are non-indexed labels
+description: Attaching metadata to logs.
+---
+# What are non-indexed labels
+
+One of the powerful features of Loki is parsing logs at query time to extract metadata and build labels out of it.
+However, the parsing of logs at query time comes with a cost which can be significantly high for, as an example,
+large json blobs or a poorly written query using complex regex patterns.
+
+In addition, the data extracted from logs at query time is usually high cardinality, which can’t be stored
+in the index without blowing the cardinality.
+
+Non-indexed labels are a way to attach metadata to logs without indexing them. Examples of useful metadata are
+trace IDs, user IDs, and any other label that is often used in queries but has high cardinality and is expensive
+to extract at query time.
+
+## Attaching non-indexed labels to log lines
+
+Non-indexed labels are optionally attached to log lines in the push payload along with each log line and the timestamp.
```

Comment:
Let's make this active voice, since it has to be configured.
```suggestion
You have the option to attach non-indexed labels to log lines in the push payload along with each log line and the timestamp.
```